thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -360,6 +352,10 @@ class CDistributorBase : public CSimpleInterface, implements IHashDistributor, i
             {
                 return bucket.getClear();
             }
+            bool queryMarkSenderFinished()
+            {
+                return (atomic_cas(&senderFinished, 1, 0));
```

Comment:
picky: this will create a warning on some compilers (e.g., windows) without a != 0